src/lib/containers/entity_finder/details/view/DetailsView.tsx

```diff
@@ -109,7 +113,12 @@ export const DetailsView: React.FunctionComponent<DetailsViewProps> = ({
 
   const [shouldSelectAll, setShouldSelectAll] = useState(false)
   const [showLoadingScreen, setShowLoadingScreen] = useState(false)
-
+  
+  const cancelQuery = () => {
+    queryClient.cancelQueries(['entitychildren',requestKey])
```

Comment:
Just want to add a comment with some additional context for the future--we don't necessarily have to cancel the current request here, but there are good reasons to do so!
```suggestion
    // It's likely that the user will be throttled by the Synapse backend and may be waiting a noticeable amount of time for the current request, so cancel it (in addition to cancelling future requests)
    queryClient.cancelQueries(['entitychildren',requestKey])
```